sdk/identity/identity/test/internal/node/clientCertificateCredential.spec.ts

```diff
@@ -43,15 +46,15 @@ describe("ClientCertificateCredential (internal)", function () {
   it("Should throw if the parameteres are not correctly specified", async function () {
     let errors: Error[] = [];
     try {
-      new ClientCertificateCredential(undefined as any, env.AZURE_CLIENT_ID, {
-        certificatePath: env.AZURE_CLIENT_CERTIFICATE_PATH,
+      new ClientCertificateCredential(undefined as any, env.AZURE_CLIENT_ID!, {
+        certificatePath: env.AZURE_CLIENT_CERTIFICATE_PATH!,
```

Comment:
you could also use `assertEnvironmentVariable` that is exported from the recorder instead of the `!`

Reply:
just FYI, not important, unless eslint plugin rules change.